bin/run.sh

```diff
@@ -81,7 +116,7 @@ else
     echo "Installed Virtualenv"
 fi
```

Comment:
I would vote for removing lines 87-117, which are a set of `which || brew` commands.

This is already handled by the Travis config, and for normal users we should document rather than script IMO.